src/proposals/forms/mixins.py

```diff
@@ -227,4 +231,8 @@ class Meta:
                 "(coronavirus) is still raging this August?"
 
             ),
+            'first_time_speaker': _(
```

Comment:
It's my fault that I didn't mention that we do not this tickbox for the tutorials. Please delete the changes related to the tutorial model.

Reply:
Ok. I have deleted the changes related to the tutorial model.